assets/js/components/exchageRates/calculator/CurrencySelector.js

```diff
@@ -0,0 +1,32 @@
+import React, {useEffect} from "react";
+
+
+export default function CurrencySelector({availableCurrencies, setSelected, selected}) {
+
+    // set first available as default
+    useEffect(() => {
+        if (!availableCurrencies) return;
+
+        setSelected(availableCurrencies[0]);
+    }, [])
+
+    let handleChange = (e) => {
+        setSelected(e.target.value);
+    }
```

Comment:
Nie jestem pewien - handleChange musi być let'em?
Jeśli tylko to możliwe powinien być constem